src/test/kotlin/com/igorwojda/string/isanagram/challenge.kt

```diff
@@ -4,7 +4,8 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun isAnagram(str1: String, str2: String): Boolean {
-    TODO("not implemented")
+    fun String.getLetters() = run { toLowerCase().filter{it.isLetter()} }
+    return str1.getLetters().groupBy { it }==str2.getLetters().groupBy { it }
```

Comment:
Code style-wise, pay attention to the operators(equality operators in this case) that require spacing before or after and those that do not( unary operators: `!`, `-`, `+`, `++`, `--` and range operator: `..`)